packages/web-api-send-notification-job-manager/src/setup-send-notification-job-manager-container.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { AzureServicesIocTypes, BatchTaskParameterProvider, registerAzureServicesToContainer } from 'azure-services';
+import { setupRuntimeConfigContainer } from 'common';
+import { Container } from 'inversify';
+import { registerGlobalLoggerToContainer } from 'logger';
+
+export function setupSendNotificationJobManagerContainer(): Container {
```

Comment:
do you wanna call it "NotificationSender" instead of "SendNotification"?

**Applies for all files with "send-notification"**

Reply:
send-notification is what we are using for the folder names for both runner & job manager. so, that way it is consistent.